src/main/java/core/basesyntax/service/impl/FileServiceCsvImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileService;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceCsvImpl implements FileService {
+    @Override
+    public List<String> readFile(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("File not found: " + fileName, e);
+        } catch (IOException e) {
+            throw new RuntimeException("Error reading file: " + fileName, e);
+        }
+    }
+
+    @Override
+    public void writeFile(String report, String fileName) {
+        File file = new File(fileName);
```

Comment:
You are creating a new File object but not using it. The FileWriter is already handling the file creation, so the File object here is unnecessary.